torchaudio/functional/__init__.py

```diff
@@ -88,4 +89,5 @@
     'vad',
     'apply_codec',
     'resample',
+    "edit_distance",
```

Comment:
Can you use single quotation for the consistency with the others?

Reply:
In `functional.py`, most of them use double quotations. so it's play to leave it as-is. (and please do not try to fix the others that's not the concern of this PR)

Reply:
This is addressed [here](https://github.com/pytorch/audio/pull/1601/files#diff-efef667849875874740a56ce4170b267e3ec6a70c91670864e6ad4e3a4202509R92).